### DISCUSSION TITLE
Updates about the project

### DISCUSSION THREAD

**Discussion by mayrinck** [mayrinck/FOSSonMicrosoftStore#33] [Announcements]
Hey there guys! It's been a little while since I've posted anything around here... I've been a lot busy lately with work/life/health and other stuff so I did no actual updates to any apps you've suggested... Now I feel that from December and forward I will be able to come back to this project and do the updates to the apps, but right now I just can't do that.

Hope you're patient enough to wait a little longer.
The project is not dead, but it's slow right now...

**Comment by JuliaAgostino** [comment]
> Hey there guys! It's been a little while since I've posted anything around here... I've been a lot busy lately with work/life/health and other stuff so I did no actual updates to any apps you've suggested... Now I feel that from December and forward I will be able to come back to this project and do the updates to the apps, but right now I just can't do that.
> 
> Hope you're patient enough to wait a little longer. The project is not dead, but it's slow right now...

Okay, we totally understand!
